pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -197,6 +198,22 @@ def test_setitem_extension_types(self, obj, dtype):
 
         tm.assert_frame_equal(df, expected)
 
+    @pytest.mark.parametrize(
+        "ea_name",
+        # property would require instantiation
+        [
+            dtype.name
+            for dtype in ea_registry.dtypes
+            if not isinstance(dtype.name, property)
+        ],
```

Comment:
Can you maybe just add a few hardcoded ones that are not covered now because of being properties, like `+ ["datetime64[ns, UTC]", "period[D]"]`, to be complete?

Reply:
Certainly, good idea. Done.